chain/gen/genesis/miners.go

```diff
@@ -369,14 +369,15 @@ func SetupStorageMiners(ctx context.Context, cs *store.ChainStore, sys vm.Syscal
 		// Commit sectors
 		{
 			for pi, preseal := range m.Sectors {
-				params := &minertypes.SectorPreCommitInfo{
+				params := &miner.PreCommitSectorBatchParams2{Sectors: []miner.SectorPreCommitInfo{{
```

Comment:
Note: making this change like so will break genesis creation for earlier actors versions (everything before v9). We probably want to gate this behind a nv  check.

Reply:
Resolving in order to fix up tests.